src/gtfs/mod.rs

```diff
@@ -23,11 +23,46 @@ use collection::CollectionWithId;
 use common_format::manage_calendars;
 use gtfs::read::EquipmentList;
 use model::{Collections, Model};
-use objects::Comment;
+use objects;
 use read_utils::add_prefix;
 use std::path::Path;
 use Result;
 
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+struct Agency {
+    #[serde(rename = "agency_id")]
+    id: Option<String>,
+    #[serde(rename = "agency_name")]
+    name: String,
+    #[serde(rename = "agency_url")]
+    url: String,
+    #[serde(rename = "agency_timezone")]
+    timezone: Option<String>,
+    #[serde(rename = "agency_lang")]
+    lang: Option<String>,
+    #[serde(rename = "agency_phone")]
+    phone: Option<String>,
+    #[serde(rename = "agency_email")]
+    email: Option<String>,
+}
+
+impl From<objects::Network> for Agency {
+    fn from(n: objects::Network) -> Agency {
+        Agency {
+            id: Some(n.id.clone()),
+            name: n.name.clone(),
+            url: n
+                .url
+                .clone()
+                .unwrap_or("http://www.navitia.io/".to_string()),
+            timezone: n.timezone.clone().or(Some("Europe/Paris".to_string())),
```

Comment:
Are you sure of these defaults? @prhod ?

Reply:
These are the actual defaults for Navitia.io, but they shouldn't be used in most cases.
Do you have a better idea ?